docs/docs/0.7.2/policies/external-services.md

```diff
@@ -0,0 +1,54 @@
+# External Service
+
+This policy enables consuming services that are not part of the mesh from services running inside it. The default passthrough cluster will allow for accessing any non-mesh host by its domain name or IP address, however as the name of the feature suggests, this pattern leaves the mesh administrator with no tools to aply any policies for such traffic. Therefore, ExternalService resource allows for declaring the desired external resource as a named service within the mesh and enabling the observability, security and traffic manipulation similar to any other service in the mesh.
+
+## The ExternalService resource
+
+A simple HTTP service can be defined as follows
+
+:::: tabs :options="{ useUrlFragment: false }"
+::: tab "Kubernetes"
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: ExternalService
+mesh: default
+metadata:
+  namespace: default
+  name: httpbin
+spec:
+  tags:
+    kuma.io/service: httpbin
+    kuma.io/protocol: http
+  networking:
+    address: httpbin.org:80
+    tls:
+      enabled: false
+```
+
+We will apply the configuration with `kubectl apply -f [..]`.
+:::
+
+::: tab "Universal"
+```yaml
+type: ExternalService
+mesh: default
+name: httpbin
+tags:
+  kuma.io/service: httpbin
+  kuma.io/protocol: http
+networking:
+  address: httpbin.org:80
+  tls:
+    enabled: false
+```
+
+We will apply the configuration with `kumactl apply -f [..]` or via the [HTTP API](/docs/0.7.2/documentation/http-api).
+:::
+:::: 
+
+ * `tags` the external service can include an arbitrary number of tags, where `kuma.io/service` is a mandatory one. The special `kuma.io/protocol` tag is also taken into account and supports the standard Kuma protocol values. It designates the specific protocol being used by that service.
```

Comment:
Can the `kuma.io/service` value be the same as the service within the mesh? If not, state this in the docs.